tutorials/shaders/compute_shaders.rst

```diff
@@ -129,8 +222,9 @@ So let's initialize an array of floats and create a storage buffer:
     // Each float has 8 byte (32 bit) so 10 x 8 = 80 bytes
     var buffer = rd.StorageBufferCreate((uint)inputBytes.Length, inputBytes);
 
-With the buffer in place we need to tell the rendering device to use this buffer.
-To do that we will need to create a uniform (like in normal shaders) and assign it to a uniform set which we can pass to our shader later.
+With the buffer in place we need to tell the rendering device to use this
+buffer. To do that we will need to create a uniform (like in normal shaders) and
+assign it to a uniform set which we can pass to our shader later.
```

Comment:
```suggestion
With the buffer in place we need to tell the rendering device to use this
buffer. To do that, we will need to create a uniform (like in normal shaders)
and assign it to a uniform set which we can pass to our shader later.
```

Reply:
A comma should not go here, it breaks the flow of the sentence